etc/workflows/offload.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>ng-offload</id>
```

Comment:
The id still has "ng-". Wouldn't something like offload-to-remote/restore-from-remote or something similar be better names? Offload is pretty clear, but 'restore' is ambiguous imho. But whatever you think is best.